kasa/cli.py

```diff
@@ -345,19 +345,22 @@ async def brightness(dev: SmartBulb, brightness: int, transition: int):
 async def temperature(dev: SmartBulb, temperature: int, transition: int):
     """Get or set color temperature."""
     await dev.update()
-    if temperature is None:
-        click.echo(f"Color temperature: {dev.color_temp}")
-        valid_temperature_range = dev.valid_temperature_range
-        if valid_temperature_range != (0, 0):
-            click.echo("(min: {}, max: {})".format(*valid_temperature_range))
+    if not dev.is_plug:
```

Comment:
It's better to fail early to avoid the extra indent, so:
```
if not dev.is_bulb:
    click.echo(f"Device is not bulb but {dev.device_type}")
    return -1

...
```

Reply:
Yeah, that makes sense. As long as it loops back to the program instead of giving an error :)

Reply:
Hmh, this isn't resolved yet, did you forget to push? :)

Reply:
Sorry, this is my first PR, lol...  
Push should be going through the checks now :)

Reply:
No worries! :+1: Looks like there was a small linting issue still. Btw, if you are planning to do more PRs, install `pre-commit` and run `pre-commit install` and you'll get the checks run automatically for commits. Alternatively, you can run `pre-commit run -a` to perform the checks without installing the hooks (that install command).

Reply:
> No worries! 👍 Looks like there was a small linting issue still. Btw, if you are planning to do more PRs, install `pre-commit` and run `pre-commit install` and you'll get the checks run automatically for commits. Alternatively, you can run `pre-commit run -a` to perform the checks without installing the hooks (that install command).

Thanks! I'll do just that.